symfony/5.4-apache/zzz-artifakt.ini

```diff
@@ -0,0 +1,25 @@
+memory_limit = 768M
```

Comment:
What is this limit ? does it collide with resource limits set in v6 ?

Reply:
it is the current limit set for all php based stacks in v4/v5 since the beginning. the same is applied in our cookbooks on gitlab too. if we are to move this configuration, we can process this in a separate and global patch later.